### DISCUSSION TITLE
Configure redirect after Form submit

### DISCUSSION THREAD

**Discussion by fuerst** [Budibase/budibase#4961] [Help] [answered]
Hi all, 

the use case I evaluate Budibase for is an event registration form. I created a public form used for adding a table row. Here I want to configure where to send public users to after they add a row to a table. I did not find a way to achieve this. I'm looking for something like "redirect to route after submitting the form". 

Is this possible somehow? Using Javascript would be fine for me, an option would be much appreciated though. Thanks a lot!

**Comment by joebudi** [comment, marked as answer]
Hi!
On the button component (for submitting), within your public form, click the button actions settings.

A binding drawer will open from the bottom.

In the binding drawer, after the save action, add a navigate to action. Then add your destination url.

And that should be you.

**Reply by fuerst** [reply]
That easy - and I did not see it. Thank you!
